lerobot/configs/policy/act_real_no_state.yaml

```diff
@@ -0,0 +1,111 @@
+# @package _global_
```

Comment:
I have a comment/question above about what makes the cut for adding a config file. I'm doubling down now, because I think this file may add more maintainability cost than it's worth.

I'm also not sure that this is the right place to discuss experiments and conclusions. After all, it could easily be that under some other reasonable set of conditions, the conclusion is different.

Again, perhaps not fussed enough to block, but let's think about this in light of our "simplicity" mantra.

Reply:
Mhmm. Let's keep an eye on it. Resolved for now.